README.md

```diff
@@ -1,4 +1,85 @@
-# Getting Started with Create React App
+# welcome React-Redux Capstone Project
+
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [welcome React-Redux Capstone Project](#welcome-react-redux-capstone-project)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Stocks Prices NASDAQ! ](#-stocks-prices-nasdaq-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo  \[Stocks Prices NASDAQ!\]](#-live-demo--stocks-prices-nasdaq)
+  - [💻 Getting Started ](#-getting-started-)
+  - [Available Scripts](#available-scripts)
+    - [`npm start`](#npm-start)
+    - [`npm test`](#npm-test)
+    - [`npm run build`](#npm-run-build)
+    - [`npm run eject`](#npm-run-eject)
+  - [Learn More](#learn-more)
+    - [Code Splitting](#code-splitting)
+    - [Analyzing the Bundle Size](#analyzing-the-bundle-size)
+    - [Making a Progressive Web App](#making-a-progressive-web-app)
+    - [Advanced Configuration](#advanced-configuration)
+    - [Deployment](#deployment)
+    - [`npm run build` fails to minify](#npm-run-build-fails-to-minify)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Stocks Prices NASDAQ! <a name="about-project"></a>
+
+**Stocks Prices NASDAQ** This React capstone project is about building a mobile web application to check a list of metrics (numeric values) that you will create making use of React and Redux.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+- React
+- Redux
+- CSS
+- NodeJS
+- Jest
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://react.dev/">React</></li>
+    <li><a href="https://react-redux.js.org/">React Redux</a></li>
+    <li><a href="hhttps://jestjs.io/">Jest</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Select an API that provides numeric data about a topic that you like and then build the webapp around it.**
+- **The webapp have several pages.**
+- **one page with a list of items that could be filtered by some parameters**
+- **one page for the item details.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"> [Stocks Prices NASDAQ!]
+
+<a href="https://stocks-prices-nasdaq.onrender.com">View page</a> 
+<br>
+<a href="https://www.loom.com/share/e564c7a4461143608507d40961d828f3?sid=82c53767-6d2e-437a-9f85-734edbaaac4c">View presentation</a>
```

Comment:
- [ ]  Great job on making your presentation but kindly make a new video presentation that contains a clear and objective description of the project in professional English. The topic and purpose of the project should made clear. highlighting all the project features.

The video contains a highlight about a piece of the code you find interesting or are proud of making. please make a new presentation video
highlighting a piece of the code you find interesting or are proud of making.
also don't forget The maximum length of the video is 5 minutes.